config/config.go

```diff
@@ -37,6 +37,8 @@ type Config struct {
 		// IsBare if true this repository is assumed to be bare and has no
 		// working directory associated with it
 		IsBare bool
+		// Worktree is the path to the root of the working tree
+		Worktree string
```

Comment:
change name to `WorkTtree`.

Add full stop at the end of the phrase.

We should call it `WorkTreePath`.